### ISSUE TITLE
[tooltip] Tooltip should be dismissable on Hover

### ISSUE THREAD

**Issue by erikkroes** [ing-bank/lion#368]
## Description

A tooltip can easily get in the way to users who zoom in a lot. Being able to dismiss the tooltip without moving the mouse can aid in this.

WCAG 1.4.13 states: 
> A mechanism is available to dismiss the additional content without moving pointer hover or keyboard focus.
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

The method to satisfy this criterion is also listed:
> Provide a mechanism to easily dismiss the additional content, such as by pressing Escape. 

## To Reproduce

1. Open the demo of the button tooltip
2. Hover over the button with your mouse
3. A tooltip appears
4. (Expected but missing behavior) Pressing escape dismisses the tooltip

**Comment by daKmoR** [comment]
did you try in a separate preview window (e.g. without the storybook UI around)?

as Esc generally does not seem to work there 🙈

**Comment by erikkroes** [comment]
Good point, but it still doesn't work I'm afraid.
https://lion-web-components.netlify.com/iframe.html?id=local-overlay-system-tooltip--button-tooltip

**Comment by malhar12** [comment]
@erikkroes @daKmoR I'd like to work on this issue if you guys are still accepting PRs for this issue.

**Comment by daKmoR** [comment]
I think that should be already fixed 🤔

can someone verify? 🙏

**Comment by gerjanvangeest** [comment]
It is not fixed for mouseover, ESC works for keyboard navigation. 

@malhar12 if you have time, please help us fixing it.

New URL: https://lion-web-components.netlify.com/?path=/story/overlays-specific-wc-tooltip--test

**Comment by daKmoR** [comment]
as hovering does not change the element selected it will result in something like this

#### how it works for keyboard:
- open page
- \> body selected
- tab, tab, tab, ...
- \> tooltip invoker selected
- tooltip appears
- hit esc
- \> event fired on invoker (as currently selected)

#### how it works with mouse
- open page
- \> body selected
- use mouse to hover over tooltip
- tooltip appears
- hit esc
- \> event fired on body (as currently selected)

## Possible solution

- add Event (capture phase) to body
- check if I (the tooltip) am open => close

**Comment by daKmoR** [comment]
issue is being worked on 💪

**Comment by daKmoR** [comment]
solved via https://github.com/ing-bank/lion/pull/542
